Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md

```diff
@@ -121,4 +121,8 @@ For the jumpbox you just created in the hub network to have access to Key vault'
 5. Select the hub virtual network for the **Virtual network** field
 6. Click on **OK** at the bottom
 
-:arrow_forward: [Deploy a Basic Workload](./08-workload.md)
+## Enable Blue Green Deployment
+
+If you want to activate the blue green deployment you can explore the [section](./11-blue-green.md)
+
+:arrow_forward: ][Deploy a Basic Workload](./08-workload.md)
```

Comment:
Remove the extra closing bracket here: ":arrow_forward: ]"

Reply:
fixed